app-shells/bash_completion/bash_completion-2.7.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="Tab-completion for the Bash shell"
+DESCRIPTION="Since v2.04, bash has allowed you to intelligently program \
+and extend its standard completion behavior to achieve complex command \
+lines with just a few keystrokes. Imagine typing ssh [Tab] and being \
+able to complete on hosts from your ~/.ssh/known_hosts files. Or typing \
+man 3 str [Tab] and getting a list of all string handling functions in \
+the UNIX manual. mount system: [Tab] would complete on all exported \
+file-systems from the host called system, while make [Tab] would complete \
+on all targets in Makefile. This project was conceived to produce \
+programmable completion routines for the most common Linux/UNIX commands, \
+reducing the amount of typing sysadmins and programmers need to do on \
+a daily basis."
+HOMEPAGE="https://github.com/scop/bash-completion"
+COPYRIGHT="2006-2017 Debian Project"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/scop/bash-completion/releases/download/2.3/bash-completion-2.3.tar.xz"
```

Comment:
I guess you meant:
```
SOURCE_URI="https://github.com/scop/bash-completion/releases/download/$portVersion/bash-completion-$portVersion.tar.xz"
```
Did you forget to commit something on your local branch?
Or you did you "forget" to kick a build :D